.github/workflows/ci.yml

```diff
@@ -1,22 +1,24 @@
-name: Node.js CI
-on: [push, pull_request]
+name: CI
+on:
+  - push
+  - pull_request
 jobs:
-  build:
+  test:
+    name: Node.js ${{ matrix.node-version }} on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        os:
+          - 'ubuntu-latest'
+          - 'macos-latest'
+          - 'windows-latest'
+        node-version:
+          - '14'
```

Comment:
Make node version into a matrix variable so we could add more than one version later if we want to. Ideally, this would support an "lts" keyword like Travis does but that's not supported yet